licenses/CeCILL-B

```diff
@@ -0,0 +1,515 @@
+
+CeCILL-B FREE SOFTWARE LICENSE AGREEMENT
```

Comment:
Did so before pushing ;-) CeCILL-B is neither available in gentoo-science nor in the tree.